Formula/openlitespeed.rb

```diff
@@ -11,19 +11,122 @@ class Openlitespeed < Formula
     sha256 "aacafddf229c07c76481f3fc86b3295ca06b41e14555b0e8a97050098701a1d2" => :mavericks
   end
 
-  option "with-debug", "Compile with support for debug log"
-  option "with-spdy", "Compile with support for SPDY module"
+  option "with-debug", "Build with support for debug log"
+  option "with-redis", "Enable redis for cache module"
+  option "with-php7", "HACK: Ugly string replacement hack to replace php5.6 into php7 on openlitespeed's `make install`"
```

Comment:
Upstream doesn't really support compiling php7 from the get go, it's either:  
1. Install and compile php5, and then use the webadmin to recompile php7
2. Or fiddle with `build_admin_php.sh` and then `make install`

This is kind of the reason of me dabbling in homebrew formula, but I don't really want to bother upstream, so let's just remove it :(